src/z3c/recipe/tag/tests.py

```diff
@@ -10,6 +10,10 @@
 def doctest_tags_recipe():
     """Test for z3c.recipe.tag
 
+        >>> write = write  # noqa: F821 (undefined name)
+        >>> system = system  # noqa: F821 (undefined name)
+        >>> cat = cat  # noqa: F821 (undefined name)
+        >>> join = join  # noqa: F821 (undefined name)
```

Comment:
You can tell flake8 about the names passed to doctests (via test.globs) by adding a 

    [flake8]
    builtins = write, system, cat, join

to setup.cfg